apps/frontend/src/components/searchBar.tsx

```diff
@@ -190,6 +209,10 @@ const SearchBar = () => {
   const isDesktop = useMediaQuery("(min-width:1240px)");
   const isSmallPage = useMediaQuery("(min-width:740px)");
```

Comment:
we should be using the theme breakpoints here instead of manually defining them

Reply:
I kept this the same because the other methods I tried were not as responsive `window.innerWidth` `screen.width` and `screen.availWidth`